### PR TITLE
fix: cordova browserify/rollup conflict

### PR DESCRIPTION
**Pull request by chmelevskij**
browserify seems to clash with rollup setup, remove browserify in favour of rollup.

If someone has OTG cable to verify would be great before merging.

Fixes:
Screenshot 2022-12-29 at 08 15 10

### CODE REVIEW

**Comment by chmelevskij** [comment]
Noticed regression on https://github.com/betaflight/betaflight-configurator/pull/3001

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
The OTG seems to be working, some tabs seems to fail, but I'm not sure if they failed before this PR.
Screenshot_20221229-125605

**Comment by chmelevskij** [comment]
@McGiverGim Which tabs are failing?

File: gulpfile.js
Comment (haslinghuis) [inline gulpfile.js:14]: This was added in #3155 - need to check if lunix debian file is still being created.

Comment (chmelevskij) [reply gulpfile.js:14]: This literally does nothing. Do you mean https://github.com/betaflight/betaflight-configurator/blob/4890ca2892d3b6642c46cb72fc5991b776d08c69/gulpfile.js#L752 is needed?

Comment (haslinghuis) [reply gulpfile.js:14]: Yes according the example in the updated library

**Comment by McGiverGim** [comment]
> @McGiverGim Which tabs are failing?

The RX tab didn't load for example. The OSD tab load but it's very ugly at least for HD but I'm pretty sure this was ugly before this PR.
I'm out for some days so I can't test more until my return.

**Comment by haslinghuis** [comment]
Receiver tab stays loading on waiting for data.

OSD is ugly on Nokia G60 but ok on Samsung Galaxy S7 (but that was before this PR)

**Comment by haslinghuis** [comment]
@chmelevskij @McGiverGim Have found the PR which causes receiver tab bug: #3097

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
